ui-particles-angular/projects/ui-particles-angular/src/scss/mat-override/mat-input.stories.ts

```diff
@@ -0,0 +1,184 @@
+            <input matInput>
+          </mat-form-field>
+          <mat-form-field appearance="outline">
+            <mat-icon svgIcon="gio:star-outline" matPrefix></mat-icon>
+            <mat-label>Simple input with icon and hint</mat-label>
+            <input matInput>
+            <mat-hint>This is a hint text</mat-hint>
+          </mat-form-field>
+          <mat-form-field appearance="outline" >
+            <mat-icon svgIcon="gio:star-outline" matPrefix></mat-icon>
+            <mat-label>Click to see control with error</mat-label>
+            <input matInput required [formControl]="errorFormControl">
+            <mat-error *ngIf="errorFormControl.hasError('required')">This is an error message</mat-error>
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Number input</mat-label>
+            <input matInput type="number">
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Color input</mat-label>
+            <input matInput type="color">
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Password input</mat-label>
+            <input matInput type="password">
+          </mat-form-field>
+
+          <mat-form-field class="example-chip-list" appearance="outline">
+            <mat-label>Multi text input</mat-label>
+            <mat-chip-list #chipList aria-label="Fruit selection">
+              <mat-chip *ngFor="let fruit of fruits" (removed)="removeFruit(fruit, fruits)">
+                {{fruit.name}}
+                <button matChipRemove>
+                  <mat-icon>cancel</mat-icon>
+                </button>
+              </mat-chip>
+              <input placeholder="New fruit..."
+                     [matChipInputFor]="chipList"
+                     (matChipInputTokenEnd)="addFruit($event, fruits)"
+              >
+            </mat-chip-list>
+          </mat-form-field>
+          <mat-form-field class="example-chip-list" appearance="outline">
+            <mat-label>Multi text input with icon prefix</mat-label>
+            <mat-icon svgIcon="gio:star-outline" matPrefix></mat-icon>
+            <mat-chip-list #chipList aria-label="Fruit selection">
+              <mat-chip *ngFor="let fruit of fruits" (removed)="removeFruit(fruit, fruits)">
+                {{fruit.name}}
+                <button matChipRemove>
+                  <mat-icon>cancel</mat-icon>
+                </button>
+              </mat-chip>
+              <input placeholder="New fruit..."
+                     [matChipInputFor]="chipList"
+                     (matChipInputTokenEnd)="addFruit($event, fruits)"
+              >
+            </mat-chip-list>
+          </mat-form-field>
+          
+          <mat-form-field appearance="outline">
+            <mat-label>Input with text prefix</mat-label>
+            <mat-label matPrefix>http://</mat-label>
```

Comment:
```suggestion
            <span matPrefix>http://</span>
```
should work better I think?